docs/command_line.md

```diff
@@ -272,9 +266,9 @@ experiments remotely from your laptop).
 
 ## How Results are Stored and Downloaded
 
-This is detailed in [Output of a tuning job](../README.md). If tuning is run
-remotely on SageMaker, results are uploaded to S3 in regular intervals. When
-the CLI is used, additional points apply:
+This is detailed in [Output of a tuning job](../README.md#launching-a-tuning-job).
```

Comment:
Cool, I did not know how to add link to specific sections.

Reply:
Haha, I found it in one of the docs you wrote, but maybe you simply copied this from elsewhere